packages/zoe/test/swingsetTests/zoe/test-zoe.js

```diff
@@ -159,8 +159,8 @@ test.serial('zoe - swapForOption - valid inputs', async t => {
 const expectedSecondPriceAuctionOkLog = [
   '=> alice, bob, carol and dave are set up',
   'Carol: The offer has been accepted. Once the contract has been completed, please check your payout',
-  '@@ schedule task for:1, currently: 0 @@',
   'Bob: The offer has been accepted. Once the contract has been completed, please check your payout',
+  '@@ schedule task for:1, currently: 0 @@',
```

Comment:
The swapping of these two lines demonstrates that these changes from using the deprecated issuer payment methods to using the deprecated helper functions instead can result in a change of the relative order in which things happen. However, because both are deprecated, this is not a concern. (Probably would not be a concern anyway.)

Reply:
Resolving